### PR TITLE
Add css styling

### PR DESCRIPTION
**Pull request by abiola-arobieke**
In this milestone:

Zeplin UI design was parsed.
CSS styling was applied to the app.
Child components were imported into another parent for styling purposes.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -62,41 +40,55 @@ ul {
 }
 
 .book-category {
-  color: #121212;
-  font-family: 'Montserrat', sans-serif;
-  font-weight: bold;
+  font-weight: bolder;
+  opacity: 0.6;
+  font-size: 0.875rem;
```
Comment (addod19) [inline src/index.css]: - [ ] Nice job implemented CSS best practices, however, there are a few repeating rules that needs to be addressed,
`font-size: 0.875em` had been repeated, we can easily create a separate class for it and apply it on the HTML element

File: src/components/BookItem.js [hunk 2]
```diff
@@ -1,49 +1,52 @@
...
           </div>
         </div>
         <div className="w-50c progress-info">
           <div className="progress-stat">Current</div>
-          <div className="chapter">Chapter 18</div>
-          <button className="btn-progress" type="button">
+          <div className="chapter txt-dark">Chapter 18</div>
+          <button className="btn-progress bg-primary txt-upper" type="button">
             Update progress
           </button>
```
Comment (addod19) [inline src/components/BookItem.js]: - [x] In the design the text on the button is on the same line, it is not on separate line as we can see i the screenshot below

Screenshot from 2023-05-26 20-31-56
Screenshot from 2023-05-26 20-31-44

File: src/index.css [hunk 3]
```diff
@@ -62,41 +40,55 @@ ul {
...
+  font-weight: 500;
 }
 
-.author {
+.author,
+.book-feedback,
+.progress-stat,
+.book-title,
+.chapter {
   font-family: 'RobotoSlab', sans-serif;
```
Comment (addod19) [inline src/index.css]: - [ ] If we want to match our design to the original design, we can improve our text by changing the `font-family: 'RobotoSlab-Light'`
Screenshot from 2023-05-26 20-42-23

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @abiola-arobieke, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Correct Git-flow followed :100: 
- [ ] README file is professional and has all relevant headers :1st_place_medal: 
- [ ] No linter errors recorded :+1: 
- [ ] CSS best practices followed, classes were used for the styling and avoided repeating rules in most part of the code :rocket: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @abiola-arobieke  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Great job resolving all the issues mentioned by the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**